### ISSUE TITLE
Heads up: PyStack_UnpackDict was removed in Python 3.9

### ISSUE THREAD

**Issue by cgohlke** [nucleic/atom#103]
Compilation of atom-0.5.1 on Python 3.9.0b1 fails with 
```
atom/src/atomlist.cpp(711): error C3861: '_PyStack_UnpackDict': identifier not found
```

https://github.com/nucleic/atom/blob/0ab56a90a6276391d05055c5d77985b8b6823b45/atom/src/atomlist.cpp#L711

Looks like `_PyStack_UnpackDict` was removed from Python's headers and made a static function in  https://github.com/python/cpython/pull/14517

**Comment by MatthieuDartiailh** [comment]
Thanks for letting me know. I will have to investigate to see what workaround I can use.

**Comment by MatthieuDartiailh** [comment]
Closed by #108
